core/tests/protractor_utils/ProfilePage.js

```diff
@@ -62,6 +67,9 @@ var ProfilePage = function() {
     expect(numInterests).toEqual(expectedInterests.length);
 
     var interestTexts = await interests.map(async function(interestElem) {
+      await waitFor.visibilityOf(
+        interestElem,
+        'InterestElm is taking too long to appear');
```

Comment:
I think you missed part of my previous review:

```suggestion
        'InterestElem is taking too long to appear');
```